src/redux/api.js

```diff
@@ -25,4 +25,16 @@ api.fetchReservations = createAsyncThunk('reservations/FETCHALL', async () => {
   return apiCall;
 });
 
+api.postReservation = createAsyncThunk('postreservation/', async (reservation) => {
+  const apiCall = await axios
+    .post(`${baseURL()}/reservations`, {
+      user_id: reservation.user_id,
+      unit_id: reservation.unit_id,
+      location: reservation.location,
```

Comment:
[optional]
Location is never updated, and it doesn't reflect on the back-end Schema and should be removed